livre/eq/render/RayCastRenderer.cpp

```diff
@@ -115,10 +115,7 @@ struct RayCastRenderer::Impl
 
         rb.getScreenCoordinates( frustum, viewport, minPos, maxPos );
 
-        const Vector2i texSize = maxPos - minPos;
-
-        const eq::PixelViewport pvp( minPos[ 0 ], minPos[ 1 ],
-                                     texSize[ 0 ], texSize[ 1 ] );
+        const eq::PixelViewport pvp( 0, 0, viewport[2], viewport[3] );
```

Comment:
At which extend should we remove stuff? For example, I feel that it is to much if we need to change the function signature... This thing is really temporar. We will fix this bricks thingy this week. We should maybe put a comment here instead of remove and reput everything ?